create-custom-map-style/README.md

```diff
@@ -0,0 +1,990 @@
+1. Maputnik is an open source visual editor for the [Mapbox Style Specification](https://www.mapbox.com/mapbox-gl-js/style-spec). At the time of writing this editor can be either used in the browser or downloaded as a binary and can run locally. We are going to use the latter option.
+2. Go to https://github.com/maputnik/editor/wiki/Maputnik-CLI and follow the instructions for installation or optionally from your terminal:
+	```
+	wget https://github.com/maputnik/editor/releases/download/latest/maputnik-darwin.zip
+	unzip maputnik-darwin.zip
+	chmod 755 maputnik
+	```
+3. Start up Maputnik 
+	```
+	./maputnik
+	```
+	And open your browser to http://localhost:8000/#12.44/40.7356/-74.0541 
+	
+4.  It would be best to start with a clean canvas. Click "Open" in the top navigation bar, then scroll down to the bottom of "Gallery Styles" and selected "Empty Style". 
+
+	![Open](media/MaputnikOpen.png)
+	*Open A Style*
+
+	![Empty](media/MaputnikEmpty.png)
+	*Empty Canvas*
+
+5. Next we need to add the Amazon Location Service API as a data source. However, in order to do this we need to use a local proxy that can handle SigV4 (TL;DR a way to provide your AWS credentials with http requests). For this we will use [tessera](https://github.com/mojodna/tessera). From a new terminal (the `maputnik` process will be occupying the other terminal) run the following: 
+	```
+	nvm use 12
+	npm i -g tessera tilejson tilelive-xray @mapbox/tilelive @mapbox/tilejson
+	```
+6. Tessera is a pluggable [map tile](https://en.wikipedia.org/wiki/Tiled_web_map) server. Using the power of the [tilelive](https://github.com/mapbox/tilelive) ecosystem, it is capable of serving and rendering map tiles from many sources. To stream data from most sources you can install the tilelive providers as global npm packages. For an Amazon Location Service source we will create and use our own module to get a basic environment set up with minimal code changes.
+7. From your terminal run 
+	```
+	mkdir tileserver 
+	cd tileserver
+	```
+8. We'll need to init a new node project in the current directory
+	```
+	npm init --yes
+	```
+9. Replace the generated `/tileserver/package.json` (DO NOT overwrite the top-level `package.json`) with the following:
+	```
+	{
+	  "name": "tileserver",
+	  "version": "1.0.0",
+	  "description": "",
+	  "main": "tilelive-minimal.js",
+	  "scripts": {},
+	  "keywords": [],
+	  "author": "",
+	  "license": "ISC",
+	  "dependencies": {
+	  	  "aws4": "^1.11.0",
+		  "debug": "^4.3.1",
+		  "request": "^2.88.2",
+		  "retry": "^0.12.0",
+		  "semver": "^7.3.5",
+		  "url-parse": "^1.5.1"
+	  },
+	  "peerDependencies": {
+		  "@mapbox/tilelive": "*",
+		  "@mapbox/tilejson": ">= 0.6.4"
+	  }
+	}
+	```
+10. Install these dependencies by running `npm install`.
+11. In your IDE create a new file called `tileserver/tilelive-minimal.js`
+12. Copy and paste the snippet below (or download) into your  `tileserver/tilelive-minimal.js` file:
```

Comment:
Can we review the content of this separately (offline)? This can be simplified a lot.